apps/notifications/admin.py

```diff
@@ -0,0 +1,125 @@
+from django.conf.urls import url
+from django.contrib import messages
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import redirect
+from django.template.response import TemplateResponse
+
+from adhocracy4.phases import models as phase_models
+
+from . import emails
+
+
+class NotifyMixin:
+    """Provides object tools and view for notifying winners.
+
+    Will sent an email notification to all followers of ideas that are winners
+    or on the short list in the last or current module.
+    """
+
+    # Required to have the extra_list_object_tools working
+    change_list_template = (
+        'admin/advocate_europe_notifications/object_tools_change_list.html'
+    )
+
+    @property
+    def extra_list_object_tools(self):
+        """ Custom extension of change list to add additional object-tools."""
+
+        module = self.latest_module
+        if module:
+            return [
+                (
+                    'notify_shortlist',
+                    'notify shortlist {}'.format(module.name)
+                ),
+                (
+                    'notify_winners',
+                    'notify winners {}'.format(module.name)
+                ),
+            ]
+        return []
+
+    @property
+    def latest_module(self):
+        phase = phase_models.Phase.objects\
+                                 .past_and_active_phases()\
+                                 .last()
+        if phase:
+            return phase.module
```

Comment:
This function implicitly returns `None` if not `phase`. Although this is absolutely working python, I don't like the idea of implicit returns.